src/framework/preferences/js/SeparatedPanelPrefsEditor.js

```diff
@@ -217,7 +257,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             var iframeWindow = that.iframe[0].contentWindow;
             that.iframeDocument = iframeWindow.document;
 
-            that.jQuery = iframeWindow.jQuery;
+            // Use the iframe's own jQuery if it's loaded, otherwise use the
+            // parent document
+
+            var iframeWindowHasJQuery = iframeWindow.jQuery !== undefined;
+            that.jQuery = iframeWindowHasJQuery ? iframeWindow.jQuery : $;
```

Comment:
Is there a case where iframeWindow.jQuery is legitimately falsey? Could we instead write this as 
```javascript
that.jQuery = iframeWindow.jQuery || $;
```